gulpfile.js

```diff
@@ -311,7 +311,15 @@ function dist_rollup() {
 
     return rollup
         .rollup({
-            input: 'src/components/init.js',
+            input: {
+                // For any new file migrated to modules add the output path
+                // in dist on the left, on the right it's input file path.
+                // If all the things used by other files are importing
+                // it with `import/export` file doesn't have to be here.
+                // I will be picked up by rollup and bundled accrodingly.
```

Comment:
It seems for some reason at some update this was rollback again :)

```suggestion
                // I will be picked up by rollup and bundled accordingly.
```